### PR TITLE
Replace TableUtils.createTable in onUpgrade with raw SQL

### LINKED ISSUE #3001: Replace createTable scripts in DatabaseHelper onUpgrade with proper table definitions
### Bug Description
Updating the mobile app to latest available test version crashes the app.

### Steps to Reproduce
1. Make sure the device has version 49.0 snapshot of Sormas under Server (https://test.sormas.netzlink.com) and logged in user (SurvOff)
2. Open the mobile app and see that the new version is offered and available to download.
3. Click on download and install the version.
4. After the installation is complete, try launching the Sormas app

### Expected Behavior
The new version of Sormas should be accessible after updating and should not crash.

### Screenshots
App crash after updatng to Version 50 0 snapshot

### System Details
* Device: Samsung S 10
* SORMAS version: update from 49.0 to 50.0 snapshot
* Android version/Browser: 10
* Server URL: https://test.sormas.netzlink.com
* User Role: SurvOff

### Additional Information
adb logs attached.
[appcrash_Version49.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/5291280/appcrash_Version49.txt)

### PR DESCRIPTION
**Pull request by faatihi**
Fixes #3001

### CODE REVIEW

**Comment by MateStrysewske** [comment]
@faatihi Thank you very much! Martin and I will review this PR in the next days, it's quite complex to test it, so it's gonna take us some time. We'll make sure that it's definitely released with the next version though.

File: sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java [hunk 1]
```diff
@@ -1711,7 +1828,26 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 
 			case 234:
 				currentVersion = 234;
-				TableUtils.createTable(connectionSource, SormasToSormasOriginInfo.class);
+
+				getDao(SormasToSormasOriginInfo.class).executeRaw(
+					"CREATE TABLE sormasToSormasOriginInfo ("
+					+ "		comment VARCHAR,"
+					+ "		organizationId VARCHAR,"
```
Comment (MartinWahnschaffe) [inline sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java:1835]: I think we did not have an organizationId here at the time of this commit.
Instead there was a string field healthDepartment

https://github.com/hzi-braunschweig/SORMAS-Project/commit/67d6e446f79263fbf59f6315db47702593d86bd7

**Review by MartinWahnschaffe — CHANGES_REQUESTED** [review]
@MateStrysewske Found two things that need to be double-checked by you, please.

@faatihi I think the two openapi files should not part of the PR.

File: sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java [hunk 2]
```diff
@@ -1449,7 +1449,45 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
...
+				getDao(Event.class).executeRaw(
+					"CREATE TABLE events ("
+					+ "		disease VARCHAR,"
+					+ "		diseaseDetails VARCHAR,"
+					+ "		endDate BIGINT,"
+					+ "		eventDesc VARCHAR,"
+					+ "		eventLocation_id BIGINT,"
+					+ "		eventStatus VARCHAR,"
+					+ "		eventType VARCHAR,"
+					+ "		externalId VARCHAR,"
```
[the remark this answers is not in the document]
Comment (faatihi) [reply sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java:1461]: Ohk

I found the `externalId` field in the `Event` class of version 1.44.0
See [Event.java](https://github.com/hzi-braunschweig/SORMAS-Project/blob/v1.44.0/sormas-app/app/src/main/java/de/symeda/sormas/app/backend/event/Event.java)

File: sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java (hunk 1 above)
Comment (faatihi) [reply sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java:1835]: https://github.com/hzi-braunschweig/SORMAS-Project/blob/38465a1a7fea70f4b830784514fb7e46843235c7/sormas-app/app/src/main/java/de/symeda/sormas/app/backend/sormastosormas/SormasToSormasOriginInfo.java#L20-L25

I found `organizationId` in version 1.50.0

See [SormasToSormasOriginInfo.java Line 23](https://github.com/hzi-braunschweig/SORMAS-Project/blob/38465a1a7fea70f4b830784514fb7e46843235c7/sormas-app/app/src/main/java/de/symeda/sormas/app/backend/sormastosormas/SormasToSormasOriginInfo.java#L20-L25)

Comment (faatihi) [reply sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java:1835]: @MartinWahnschaffe I hope you understand my response.

During the implementation of this fix,
I went through the tagged branches to extract all change points and their app version.

I then downloaded the releases of affected versions
and installed them one after the other.
For each installation, I debug and extract the sqlite db file.
I then inspect affected tables and extract their schema into raw sql statements.

In short, I didn't approach it the way you do with commit inspection

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
The first CREATE TABLE expression does not run through. I suspect it's because of line 1482 (which should come at the end (preferably) or at least have a comma after the brackets.

**Comment by MateStrysewske** [comment]
Apart from that, I was able to upgrade the mobile app from 1.43.0 to feature branch version successfully. However, I couldn't test this with data already on the tablet because my server refused to start with 1.43.0, so I did not log in prior to upgrading to the latest version.

**Comment by faatihi** [comment]
Thanks for spotting line 1482
I have re-ordered the columns to rectify the error
I think a rebase I did might have moved things around

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
Can you please make sure that the swagger files are not part of this PR?

**Comment by faatihi** [comment]
Kindly guide me through excluding that commit
It seems to appear when I make a PR

**Comment by faatihi** [comment]
PR #5384 seems to pass all checks
Kindly see if you can merge it

I had to create a new branch and manually redo all changes
I also did not use rebase this time

The [GitHub Actions] commit still shows up but I hope it wouldn't be a problem
If I still have to remove that commit, I may have to use git's "Revert Commit"
